configs/internal_flash_no_rot.json

```diff
@@ -105,6 +105,17 @@
         },
         "NUCLEO_F429ZI": {
             "kvstore-size": "2*64*1024"
+        },
+        "DISCO_F746NG": {
+            "mbed-bootloader.bootloader-size"          : "(32*1024)",
+            "kvstore-size"                             : "(2*32*1024)",
+            "update-client.application-details"        : "(MBED_ROM_START + MBED_BOOTLOADER_SIZE + KVSTORE_SIZE)",
+            "update-client.storage-address"            : "(MBED_ROM_START+512*1024)",
+            "update-client.storage-size"               : "(512*1024)",
+            "update-client.storage-locations"          : 1,
+            "update-client.storage-page"               : 1,
+            "mbed-bootloader.max-application-size"     : "((32+127+256)*1024)",
+            "target.device_has_remove"                 : [ "LPTICKER" ]
```

Comment:
Why do you remove LPTICKER ?

@LMESTM

Reply:
This saves about 3 kB of flash (sometimes), but breaks external storage drivers.
So can only be accepted into the `internal_flash` configuration.

Reply:
This is targeting the internal_flash configuration (in this case).
That is the default I would from product point of view aim always - that allows the smallest and cheapest bill of materials. Not possible with all boards, though.